src/CollAction/Views/Projects/Find.cshtml

```diff
@@ -9,118 +9,7 @@
     var defaultDisplayModel = FindProjectViewModel.DefaultDisplayProjectViewModel;
 }
 
-<h2>@Localizer["Find"]</h2>
```

Comment:
Hmm, this deletes most of the Find page. Is this intended. In the other commits, you replace it with different html.